vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/create/ProjectCreationDialog.java

```diff
@@ -0,0 +1,375 @@
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.value.ValueChangeMode;
+import com.vaadin.flow.spring.annotation.SpringComponent;
+import com.vaadin.flow.spring.annotation.UIScope;
+import java.io.Serial;
+import java.util.ArrayList;
+import java.util.List;
+import life.qbic.datamanager.views.events.UserCancelEvent;
+import life.qbic.projectmanagement.application.ExperimentalDesignSearchService;
+import life.qbic.projectmanagement.domain.finances.offer.Offer;
+import life.qbic.projectmanagement.domain.finances.offer.OfferPreview;
+import life.qbic.projectmanagement.domain.project.ExperimentalDesignDescription;
+import life.qbic.projectmanagement.domain.project.PersonReference;
+import life.qbic.projectmanagement.domain.project.ProjectCode;
+import life.qbic.projectmanagement.domain.project.ProjectObjective;
+import life.qbic.projectmanagement.domain.project.ProjectTitle;
+
+/**
+ * <b>Project Creation Dialog</b>
+ *
+ * <p>Dialog to create a project based on a project intent</p>
+ *
+ * @since 1.0.0
+ */
+@SpringComponent
+@UIScope
+public class ProjectCreationDialog extends Dialog {
+
+  @Serial
+  private static final long serialVersionUID = 6132538769263078943L;
+  public final ComboBox<OfferPreview> offerSearchField = new ComboBox<>("Offer");
+  private HorizontalLayout codeAndTitleLayout = new HorizontalLayout();
+  private final TextField codeField = new TextField("Code");
+  private final Button generateCodeButton = new Button(new Icon(VaadinIcon.REFRESH));
+  private final FormLayout formLayout = new FormLayout();
+  private final TextField titleField = new TextField("Title");
+  private final TextArea projectObjective = new TextArea("Objective");
+  private final DefineExperimentComponent defineExperimentComponent;
+  private final VerticalLayout projectContactsLayout = new VerticalLayout();
+  public final ComboBox<PersonReference> principalInvestigator = new ComboBox<>(
+      "Principal Investigator");
+  public final ComboBox<PersonReference> responsiblePerson = new ComboBox<>(
+      "Project Responsible (optional)");
+  public final ComboBox<PersonReference> projectManager = new ComboBox<>("Project Manager");
+  private final Button createButton = new Button("Create");
+  private final Button cancelButton = new Button("Cancel");
+  private final Handler handler;
+
+  public ProjectCreationDialog(ExperimentalDesignSearchService experimentalDesignSearchService) {
+
+    initCodeAndTitleLayout();
+    projectObjective.setRequired(true);
+
+    defineExperimentComponent = new DefineExperimentComponent(experimentalDesignSearchService);
+    defineExperimentComponent.hideNameField();
```

Comment:
why hide it?

Reply:
does the first experiment have no name?

Reply:
Yes, that's the actual reason for the hide method in the first place (which also looks contrived to me and i'm very happy to hear an alternate solution). For now we don't want the user to have the possibility an experiment name during project creation according to the prototype